build-tools/packages/build-tools/src/buildVersion/buildVersion.ts

```diff
@@ -17,9 +17,16 @@
  *      The computed version output to the console.
  */
 
+import {strict as assert } from "assert";
+import {
+    isInternalVersionScheme,
+    getLatestReleaseFromList,
+    detectVersionScheme,
+} from "@fluid-tools/version-tools";
 import child_process from "child_process";
 import fs from "fs";
-import { sort as sort_semver, gt as gt_semver, prerelease as prerelease_semver } from "semver";
+import { sort as sort_semver, gte as gte_semver, prerelease as prerelease_semver } from "semver";
```

Comment:
Nit: since this is typescript code, should we use `camelCase` for function aliases?

```suggestion
import { sort as sortSemver, gte as gteSemver, prerelease as prereleaseSemver } from "semver";
```